### ISSUE TITLE
Lacey collides with Kent in the SVE Spirit's Eve event

### ISSUE THREAD

**Issue by Rupurudu** [ichortower/HatMouseLacey#2]
In SVE there are two Spirit's Eve maps. This is the second one, happens on the even years.

![Image](https://github.com/ichortower/HatMouseLacey/assets/3107676/5137d8c1-de72-4bba-802f-32adc2d0bce6)

**Comment by ichortower** [comment]
Thanks for reporting this! Hopefully I can find a better spot easily.

What I'm really interested in from your screenshot, though, is Lacey using the wrong (vanilla) breathing rectangle. Can you do me a huge favor and run `harmony_summary` in your SMAPI console, then attach your SMAPI log from that session? (or upload it to [https://smapi.io/log](https://smapi.io/log) and paste the link)? I suspect someone else is prefix patching NPC.draw and preventing my patch from executing.

**Comment by Rupurudu** [comment]
Yeah, there are two mods patching `StardewValley.NPC.draw` in my modpack.

```
   StardewValley.NPC.draw
      - ichortower.HatMouseLacey.Core (prefix)
      - spacechase0.SpaceCore (prefix)
  ```

Full log: https://smapi.io/log/13cdf38d67b94b9cac76dada706f9e1b?renew=True

Edit: Just tested, without the SpaceCore, Lacey breathes differently.

**Comment by ichortower** [comment]
Thanks for reporting this. I moved Lacey one tile down in this map setup, which is now published in 1.1.2.
(The breathing issue can't really be solved here; I'll need cooperation from spacechase0)
